### PR TITLE
Add passwordless sudo capability for Jenkins user

### PR DESCRIPTION
**Pull request by Kaitou786**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[link](https://examle.com)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* referenced pull request 1
* referenced pull request 2
* referenced pull request 2
etc. 

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: jenkins-slaves/ansible/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@
           'htop', 'screen', 'lsof', 'bash-completion', 'bzip2', 'rng-tools',
           'ansible', 'zip', 'python2-pip', 'jq', 'libXScrnSaver', 'python3', 'python3-devel',
           'python3-pip', 'glibc-devel', 'zlib-devel', 'podman', 'gcc', 'krb5-devel', 'libstdc++-static', 
-          'java-1.8.0-openjdk-devel', 'java-11-openjdk-devel']
+          'java-1.8.0-openjdk-devel', 'java-11-openjdk-devel', 'sudo']
```
Comment (winklerm) [inline jenkins-slaves/ansible/roles/common/tasks/main.yml:24]: Is sudo package really needed?

**Review by winklerm — COMMENTED** [review]
Looks good to me, just not sure if you really need to install 'sudo' package?

File: jenkins-slaves/ansible/roles/common/tasks/main.yml (hunk 1 above)
Comment (Kaitou786) [reply jenkins-slaves/ansible/roles/common/tasks/main.yml:24]: Yeah you are right, it is already there :)

**Review by winklerm — APPROVED** [review]
Thanks, looks good to me!

**Review by mbiarnes — APPROVED** [review]
LGTM

**Pull request merged**